Dockerfile

```diff
@@ -15,24 +15,24 @@
 #   docker build -t dlmonte .
 #
 
-FROM python:3.8-slim-buster
+FROM alpine:latest
```

Comment:
Replacing the base image was not strictly necessary. The reason I replaced it was because I found that the Alpine base image gave less vulnerabilities than the Debian base image. I had hoped that using Alpine would eliminate critical vulnerabilities. Unfortunately this was naive of me. However the situation with Alpine proved better than Debian, so I left the base image as Alpine.